src/main/java/me/karakelley/tictactoe/HumanUserInterface.java

```diff
@@ -8,8 +8,8 @@
 
   private static final String tie = "It's a tie!\n";
   private static final String welcome = "Welcome to Tic Tac Toe!\n";
-  private static final String player1 = "Player one";
-  private static final String player2 = "Player two";
+  private String player1Wins = "Player one wins!\n";
```

Comment:
similar consistency for naming. Constants should be named all capital snake case:

e.g. `TIE`, `PLAYER_1_WINS`

... and private static final

Reply:
Just noticed you have setters for the playerWin messages. Still, the default message can be extracted in a constant